gymnasdicts/base.py

```diff
@@ -62,13 +62,14 @@ def _select(
         else:
             raise ValueError("unexpected payload type")
 
-    _select(
-        payload,
-        {
-            key: tuple(map(strip_index, value.split(".")[1:]))
-            for key, value in pointers.items()
-        },
-    )
+    for payload in payloads:
```

Comment:
i think tat this is valid because this methods job is to flatten the payload into a list of dicts anyway